Automattic/wp-calypso
# Block Editor: Update reusable blocks list ellipsis menu

#### Changes proposed in this Pull Request

* Add a new ellipsis menu to be displayed in the Reusable Block (`wp_block`) custom post type list, that only allows to edit or trash the item.

In a follow up, we should also bring the "Import/Export as JSON" feature for Reusable Blocks, currently only available in wp-admin.

#### Note

Calypso, contrarily to wp-admin, doesn't show titles for lists, relying on highlighting the current page in the sidebar.
Reusable Blocks don't have any items in the sidebar (consistently with wp-admin), so it can be unclear what the list is.
There are two solution here: either show a Blocks item in the sidebar, or show a title.
Either way, I think we need a design point of view.

#### Testing instructions



* Open the Block Editor.
* Insert a block, open its options dropdown and select "Add to Reusable Blocks".
* Open the Reusable Blocks list at `/types/wp_block/:siteFragment` (this is currently only accessible via URL, but see D26034-code for updates).
* Click the ellipsis menu for any item.
* Make sure it only shows Edit and Trash, and that both links work as expected.
* Check for regressions on the ellipsis menu for posts and other custom post types.

Fixes #27374
